docs/sources/mimir/migration-guide/migrate-from-cortex.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Migrate from Cortex to Grafana Mimir"
+menuTitle: "Migrate from Cortex"
+description: "Learn how to migrate your deployment of Cortex to Grafana Mimir to simplify the deployment and continued operation of a horizontally scalable, multi-tenant time series database with long-term storage."
+weight: 10
+---
+
+# Migrate from Cortex to Grafana Mimir
+
+This document guides an operator through the process of migrating a Jsonnet deployment of [Cortex](https://cortexmetrics.io/) to Grafana Mimir.
+This document includes an overview of the steps required for any environment, and specific instructions for [environments deployed with Jsonnet](#migrating-to-grafana-mimir-using-jsonnet).
+This document was tested with Cortex versions 1.10 and 1.11. While it may work with more recent versions of Cortex, we cannot guarantee that.
+To migrate a Helm deployment of Cortex refer to [Migrate from Cortex](/docs/helm-charts/mimir-distributed/latest/migration-guides/migrate-from-cortex).
+
+Grafana Mimir includes significant changes that simplify the deployment and continued operation of a horizontally scalable, multi-tenant time series database with long-term storage.
+
+The changes focus on making Grafana Mimir easier to run out of the box, including:
+
+- Removing configuration parameters that don't require tuning
+- Renaming some parameters so that they're more easily understood
+- Updating the default values of some existing parameters
+
+The `mimirtool` automates configuration conversion.
+It provides a simple migration by generating Mimir configuration from Cortex configuration.
+
+## Before you begin
+
+- Ensure that you are running either Cortex 1.10.X or Cortex 1.11.X.
+
+  If you are running an older version of Cortex, upgrade to [1.11.1](https://github.com/cortexproject/cortex/releases) before proceeding with the migration.
+
+- Ensure you have installed Cortex alerting and recording rules as well as Cortex dashboards.
+
+  The monitoring mixin has both alerting and recording rules to install in either Prometheus or Cortex as well as dashboards to install in Grafana.
```

Comment:
```suggestion
  For the monitoring mixin, you need to install both alerting and recording rules in either Prometheus or Cortex. You also need to install dashboards in Grafana.
```

Reply:
Please vet this for technical accuracy. Does the monitoring mixin install alerting rules and recording rules, or does the use need to figure that out? Does my change make sense or have I misunderstood this?

If my change is technically inaccurate, let’s revisit the initial verbiage so we can make it clearer who or what is doing what.

Reply:
Does this mean, "The monitoring mixin contains both alerting and recording rules that you {can | need to} install in either Prometheus or Cortex. The mixin also contains dashboards that you {can | need to} install in Grafana."

If I have a choice, why would I make one choice or the other wrt {can | need to}?

Reply:
LGTM, thanks!